src/main/java/homeWorkPart1/task4/Main.java

```diff
@@ -1,4 +1,4 @@
-package Task4;
+package homeWorkPart1.task4;
 
 import java.text.DecimalFormat;
 
```

Comment:
Why you have used `DecimalFormat`? Do you really need it?

Reply:
Again, remember to store your objects in an instance of the superclass or interface to follow polymorphism